go/src/koding/klient/machine/index/index.go

```diff
@@ -64,6 +64,11 @@ func NewIndexFiles(root string) (*Index, error) {
 			return nil
 		}
 
+		// Don't include OSX trashes to scanned files.
+		if info.Name() == ".Trash" && info.Mode().IsDir() && runtime.GOOS == "darwin" {
```

Comment:
@rjeczalik I added this as a workaround for failing tests. Will move mount/skipper.go logic to index which will be more generic solution. But didn't want to do this in this PR.